CMakeLists.txt

```diff
@@ -13,4 +13,5 @@ add_executable(wibo
     user32.cpp
     version.cpp
     )
+target_link_libraries(wibo PRIVATE stdc++fs)
```

Comment:
This is not needed on modern compilers, and will cause failures when trying to use clang with libc++ (the equivalent for libc++ would be -lc++fs). At the very least this should be conditional on the result of a try_compile. This should be reverted.

Reply:
See e.g. https://github.com/vector-of-bool/CMakeCM/blob/6cb8a6ada09eb40c271d2f83a032e03cd4005fa3/modules/FindFilesystem.cmake for a way to properly detect this.

Reply:
whoops okay. thanks